apps/angularlogos/src/app/search/search.component.scss

```diff
@@ -1,8 +1,24 @@
 :host {
   display: block;
-  margin-top: 3em;
+  margin: 3rem 0 1rem;
 }
 
 mat-form-field {
-  width: 100%;
+  margin: 0 5%;
+  width: 90%;
+  color: #c3002f;
+  border-radius: 0.3rem;
+  box-shadow: 0 0.5rem 1rem rgba(0, 0, 0, 0.15);
+
+  mat-label {
+    color: #c3002f;
+  }
+}
+
+::ng-deep .mat-form-field-underline {
```

Comment:
`ng-deep` will be deprecated and we should not use it anymore

> The shadow-piercing descendant combinator is deprecated and support is being removed from major browsers and tools. As such we plan to drop support in Angular (for all 3 of /deep/, >>> and ::ng-deep). Until then ::ng-deep should be preferred for a broader compatibility with the tools.

https://angular.io/guide/component-styles#deprecated-deep--and-ng-deep

Reply:
Good to know!
For now I don’t really see any other alternative though 😕